missilewars-plugin/src/main/java/de/butzlabben/missilewars/configuration/Lobby.java

```diff
@@ -90,4 +93,20 @@ public List<Arena> getArenas() {
                 .map(Optional::get)
                 .collect(Collectors.toList());
     }
+
+    public void updateConfig() {
+        try {
+            Serializer.serialize(file, this);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public Location getAreaMinLocation() {
+        return new Location(getBukkitWorld(), area.getMinX(), area.getMinY(), area.getMinZ());
+    }
+
+    public Location getAreaMaxLocation() {
+        return new Location(getBukkitWorld(), area.getMaxX(), area.getMaxY(), area.getMaxZ());
+    }
```

Comment:
**What I changed:**
Currently, not used. But perhaps in the future it will be possible to define the arena of the lobby.